### DISCUSSION TITLE
How the hell do you install it on windows

### DISCUSSION THREAD

**Discussion by RealMrCactus** [prometheus-lua/Prometheus#24] [Q&A] [answered]
the exe crashes and i dont know where the lua 5.1 shit goes

**Comment by Brohammer5** [comment, marked as answer]
Hello, This can answer your question:
Just put this in command prompt
```cd C:\Users\YourName\Downloads\prometheus-windows```
then
```prometheus.exe --preset Medium yourfile.lua```

Or look at: https://github.com/levno-710/Prometheus/issues/27#issuecomment-1043464982
